### DISCUSSION TITLE
Gopro Quick - technical question

### DISCUSSION THREAD

**Discussion by Pax709** [gopro/labs#894] [General]
Hello, I would like to ask why the Qucik application when exporting always assigns a Rec2020 color space, while the originals have a Rec709 space, it does not matter 10 or 8 bit, 5.3 or 4K. Is the reason that the developers do not use Windows and do not see a problem with it ?

**Comment by dnewman-gpsw** [comment]
There are Windows developers, including me.  Rendering from Android or iOS or MacOS?  I don't use this tool. I did try and see issues with Android.

**Reply by Pax709** [reply]
Yes, this is Android, on the phone Pixel 8 Quick assigns the space Rec2020 and on the Samsung Galaxy Ultra Rec 601, on a daily basis I use only Davinci Resolve, but in my spare time I wanted to edit a short video and such a surprise. David are you able to write what is the reason, is it GPU or Android ?

**Reply by dnewman-gpsw** [reply]
I'm not involved with mobile development, so no idea.  I also tested with Pixel 8.

**Reply by Pax709** [reply]
I reported a continuation of the problem and the matter is already being investigated, there is a problem only with files above 1440p, that is, 4 and 5.3k is the problem, on the Pixel 8 Quick assigns the space Rec2020 and on the Samsung S23Ultra Rec601 and I know they are testing the Quick app on the Samsungs. and on the Samsung the problem may not be noticeable, too little difference between Rec601 vs Rec709.

**Reply by dnewman-gpsw** [reply]
I heard the same thing, it is under investigation.
